stylesheets/commons/Colours.less

```diff
@@ -35,7 +35,7 @@ Besides these twelve there is Gray which will fit in with them very well. There
 .gray-theme-light-alt-bg,
 ul.nav-tabs li.game-none,
 .overview.game-none {
-	background-color: var( --clr-moon-80, #d3d3d3 );
+	background-color: var( --clr-moon-80, #ababab );
```

Comment:
```suggestion
	background-color: var( --clr-moon-80, #d3d3d3 );
}
[ data-darkreader-scheme="dark" ] &,
.theme--dark & {
	.overview.game-none {
		background: var( --clr-moon-20 );
	}
}
```
you probably want something like this
possibly with 30 instead of 20
i suggest testing it with dev tools in browser